.vscode/launch.json

```diff
@@ -0,0 +1,364 @@
+{
```

Comment:
Please remove the .vscode folder from the PR.

Reply:
yes done